### DISCUSSION TITLE
关于工作流的英文翻译？

### DISCUSSION THREAD

**Discussion by gaoweiwei53** [apache/dolphinscheduler#12637] [Q&A] [answered]
为什么官网和web界面上关于工作流的英文翻译有的地方是workflow，有的地方是process？

**Comment by DarkAssassinator** [comment, marked as answer]
i think that we need change to workflow, WDYT. cc @ruanwenjun @caishunfeng @SbloodyS @EricGao888

**Reply by EricGao888** [reply]
@DarkAssassinator  
* https://github.com/apache/dolphinscheduler/pull/12297 We have a PR for this, but seems no more update. Of course it would be better that we change `process` to `workflow`

**Reply by caishunfeng** [reply]
+1, we should use workflow but not process.

**Reply by SbloodyS** [reply]
+1

**Reply by ruanwenjun** [reply]
+1
